static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -75,10 +80,23 @@ export class AccessMethod extends Component {
       )
     } = selectedMethod || {}
 
+    const isHarmony = selectedAccessMethod
+      ? selectedAccessMethod.startsWith('harmony')
+      : false
+
+    let selectedHarmonyMethodName = ''
+    if (selectedAccessMethod
+        && selectedAccessMethod.startsWith('harmony')
```

Comment:
```suggestion
        && isHarmony
```
isn't this already being computed above?